core/src/main/java/org/elasticsearch/index/query/CommonTermsQueryBuilder.java

```diff
@@ -222,4 +258,135 @@ public void doXContent(XContentBuilder builder, Params params) throws IOExceptio
     public String getName() {
         return NAME;
     }
+
+    @Override
+    public Query toQuery(QueryParseContext parseContext) throws QueryParsingException, IOException {
+        String field;
+        MappedFieldType mapper = parseContext.fieldMapper(fieldName);
```

Comment:
in master we call this fieldType rather than mapper, I find it a better name, do you mind changing it?